README.md

```diff
@@ -54,46 +54,66 @@ We recommend that you link to `<url for repository's GitHub pages>/workshop/HOME
 Use the `Run workflow` drop down menu; you will be required to input the appropriate `training-modules` release tag for your training workshop.
 
-## Local development 
+## Local development
 
-It can be helpful to build the GitHub pages site locally to check that passing parameter values is working as expected. 
-The following instructions are for Mac OS and have only been used on Mojave. 
+It can be helpful to build the GitHub pages site locally to check that passing parameter values is working as expected.
 
-1. Make sure you have [Bundler](https://bundler.io/) installed. Installation can be accomplished with `gem install bundler`.
- 
-  If you encounter a permissions error, it was solved (using this StackOverflow post as a guide) with the following on Mojave with [Homebrew](https://brew.sh/):
+### Installing GitHub pages dependencies locally
 
-  ```
-  # install rbenv
-  brew update
-  brew install rbenv
+Installing the dependencies for GitHub pages is best done in a separate ruby environment, managed by [rbenv](https://github.com/rbenv/) and [Bundler](https://bundler.io)
 
-  # add to ~/.bash_profile
-  echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
-  ```
+The following instructions are for installation on Mac OS.
+1. [Install rbenv](https://github.com/rbenv/rbenv#installation).
+The easiest, and recommended, installation is through Homebrew:
+```
+# install rbenv
+brew update
+brew install rbenv ruby-build
+```
+Alternatively, you can install with [rbenv-installer](https://github.com/rbenv/rbenv-installer#rbenv-installer) which will work even without Homebrew (though it uses Homebrew if you have it).
 
-  Start a new shell to initialize `rbenv`.
+2. Initialize rbenv and install it for your shell:
+```
+ # run this and follow the printed instructions:
+rbenv init
+```
+Start a new shell to initialize `rbenv`.
 
+3. Install a current stable ruby version
+  ```
+  # install and set up v3.1.3 of Ruby
+  rbenv install 3.1.3
   ```
-  # install and setup v2.7.1 of Ruby
-  rbenv install 2.7.1
-  rbenv shell 2.7.1
-  rbenv rehash
-  ``` 
 
-  Now you're ready to run `gem install bundler`. 
+4. Activate your ruby version. Here you have a few options:
 
-2. Install Jekyll (`gem install jekyll`).
-3. Run `bundle install` from the root of this repository, where the `Gemfile` is located.
-4. Run `bundle exec jekyll serve` and navigate to the server address.
+   a. If you want to set the global ruby version that will be used wherever you use `ruby`, you can use
+   ```
+   rbenv global 3.1.3
+   ```
 
-If you've already followed the steps above and are in a new shell, run the following steps to build locally:
+   b. To only use this version of ruby within the repository only, navigate to the root of this repository and use:
+   ```
+   rbenv local 3.1.3
+   ```
+   This will create a `.ruby-version` file that will automatically activate this version of ruby whenever you are in this directory.
 
+5. Now you should be ready to install Bundler:
```

Comment:
Before this step you need to start a new shell for the `ruby` version to kick in, so the right `gem` executable is found. Otherwise my `gem` was `/usr/bin/gem`. In a new shell, `which gem` gave `/Users/sjspielman/.rbenv/shims/gem` which is the right one!

Reply:
I would expect this to happen after the first shell restart. After the `rbenv init` instructions are followed (the `eval` line added to PATH), the changes should be instant.